packages/ddp-server/livedata_server.js

```diff
@@ -1224,9 +1224,9 @@ _.extend(Subscription.prototype, {
   onStop: function (callback) {
     var self = this;
     if (self._isDeactivated())
-      callback();
+      callback.call(self);
     else
-      self._stopCallbacks.push(callback);
+      self._stopCallbacks.push(Meteor.bindEnvironment(callback.bind(self)));
```

Comment:
I did not know that it took context as the last argument, I'll use that instead of `bind`.
I didn't know it was smart enough to figure that out so I took the safer option, but I'll up that and skip the `call`/`bind` :+1: